rpcq/_base.py

```diff
@@ -120,13 +120,12 @@ def types():
         :return: A dictionary of ``Message`` types.
         :rtype: Dict[str,type]
         """
-        if Message._types is None:
-            Message._types = {}
```

Comment:
One thread might execute this while another thread is in the middle of building the dictionary. The dictionary is a class member and eventually contains a union of all message types encountered, so this fix should cause no backwards incompatibilities.